iot_config/src/org_service.rs

```diff
@@ -84,7 +84,10 @@ impl iot_config::Org for OrgService {
 
         let org = org::get_with_constraints(request.oui, &self.pool)
             .await
-            .map_err(|_| Status::internal("org get failed"))?;
+            .map_err(|err| {
```

Comment:
might be helpful to output some of the request details ( in this case oui ) in the error msg, i see you are doing that in some but not others

Reply:
agreed; i've tried to add additional context wherever i saw a gap and shifted as many things over to using the structured logging k/v metadata pairs in front of the log string wherever i could without introducing a need to derive `tracing::Value` on custom structs; we can go further with that as needed in future